pkgs/tools/package-management/nix-upfetch/default.nix

```diff
@@ -0,0 +1,102 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, rustPlatform, pkgconfig, ncurses
+, asciidoc, docbook_xml_dtd_45, docbook_xsl, libxml2, libxslt
+, libredirect, coreutils, gawk, gnugrep, gnused, jq, nix, nix-prefetch }:
+
+with stdenv.lib;
+
+let
+  nix-update-fetch = rustPlatform.buildRustPackage rec {
+    name = "${pname}-${version}";
+    pname = "nix-update-fetch";
+    version = "0.1.0";
+
+    src = fetchFromGitHub {
+      owner = "msteen";
+      repo = "nix-update-fetch";
+      rev = "bc201ec28be669f67501058e97b93e6a6e7b4e07";
+      sha256 = "09qqk9wcbbnwv0rkmr18k4pvaf6xpbrfci7ki44v8m6p4pwvqczw";
+    };
+
+    RUSTC_BOOTSTRAP = 1;
+
+    buildInputs = [ pkgconfig ncurses ];
+
+    cargoSha256 = "0g2gmmhx2gcb02yqmzavx7fqyvdblgg16rhq10rw2slnrmsz84k6";
+
+    meta = {
+      description = "Update a call made to a fetcher function call and its surrounding bindings";
+      homepage = https://github.com/msteen/nix-update-fetch;
+      license = licenses.mit;
+      maintainers = with maintainers; [ msteen ];
+      platforms = platforms.all;
+    };
+  };
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "nix-upfetch";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "msteen";
+    repo = "nix-upfetch";
+    rev = "dd133c6a345710a285d5ed90c573b5b974e65c96";
+    sha256 = "1kmsr2j2qybrb8ksh5vq2zmx54gbsc56vcd6gs7jhswfmzhw0blc";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    asciidoc docbook_xml_dtd_45 docbook_xsl libxml2 libxslt
+  ];
+
+  configurePhase = ''
+    . configure.sh
```

Comment:
Does [`configureScript`] (https://github.com/NixOS/nixpkgs/blob/bccab1610935329962b80259b60d3b2d347b69d4/pkgs/stdenv/generic/setup.sh#L1002) not work?

Reply:
It probably will, simply did not know about it, thanks for the suggestion!

Reply:
Tried it, but since it is simply setting a variable, turning it into a configure scripts complicates matters unnecessary, so I am keeping it as it is for now, unless I find some better way.